tests/rpmquery.at

```diff
@@ -647,7 +647,7 @@ AT_SETUP([xml format])
 AT_KEYWORDS([query])
 RPMTEST_CHECK([
 RPMDB_INIT
-runroot rpm -qp --xml  data/RPMS/hello-2.0-1.x86_64.rpm
+runroot rpm -qp --xml  /data/RPMS/hello-2.0-1.x86_64.rpm
```

Comment:
These relative -> absolute path changes look like something that could be applied regardless of this change. Most path usage in the tests is absolute anyhow, these mostly just look out of the place now, but could be remnants from the days where we actually ran some stuff outside the test-root.

Reply:
Indeed, this is another example of something that could've gone in separately. In fact, I'll start submitting these little changes now, before this PR is undrafted.